web/src/peer_data.ts

```diff
@@ -76,39 +77,39 @@ export function potential_subscribers(stream_id) {
     return people.filter_all_users(is_potential_subscriber);
 }
 
-export function get_subscriber_count(stream_id) {
+export function get_subscriber_count(stream_id: number): number {
     const subscribers = get_user_set(stream_id);
     return subscribers.size;
 }
 
-export function get_subscribers(stream_id) {
+export function get_subscribers(stream_id: number): number[] {
     // This is our external interface for callers who just
     // want an array of user_ids who are subscribed to a stream.
     const subscribers = get_user_set(stream_id);
 
     return [...subscribers.keys()];
 }
 
-export function set_subscribers(stream_id, user_ids) {
+export function set_subscribers(stream_id: number, user_ids: number[]): void {
     const subscribers = new LazySet(user_ids || []);
```

Comment:
`user_ids || []` would be redundant `user_ids` is always guaranteed to be `number[]`. We can just use `user_ids` directly.